students/20th/sunkyungnoh/login/login.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const loginForm = document.querySelector('.loginForm');
+const inputLogin = document.querySelectorAll('input');
+const inputID = document.querySelectorAll('input')[0];
+const inputPW = document.querySelectorAll('input')[1];
+const loginBtn = document.querySelector('.login.btn');
+
+function moveToUpside(input) {
+    const descriptionType = input.previousElementSibling
+
+    if (input.value) {
+        descriptionType.classList.add('focused');
+        input.classList.add('entered');
+    } else {
+        descriptionType.classList.remove('focused');
+        input.classList.remove('entered');
+    }
+};
+
+function activeBtn() {
+    if (inputID.value.includes('@') && inputPW.value.length >= 6) {
+        loginBtn.removeAttribute('disabled');
+        loginBtn.classList.add('active');  
+    } else {
+        loginBtn.setAttribute('disabled', 'true');
+        loginBtn.classList.remove('active')
+    }
+};
+
+const login = () => {
+    inputLogin.forEach(input => {
+        input.addEventListener('input', activeBtn)
+        input.addEventListener('input', () => moveToUpside(input))
+    })
+};
+
+login();
```

Comment:
기왕 Form 을 사용하고 있으니, Form 에 이벤트 리스너를 할당해보세요! 두개 input 에 대한 이벤트가 한번에 잡히게 됩니다.
```suggestion
loginForm.addEventListener('input', (e) => {
  enableLogIn();
  moveLabelUp(e.target);
});
```

Reply:
오우! 훨씬 간단하게 할 수 있군요!